java/src/jmri/jmrix/srcp/parser/package-info.java

```diff
@@ -7,6 +7,6 @@
  */
 @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value={"DLS_DEAD_LOCAL_STORE", "EI_EXPOSE_REP2", "MS_OOI_PKGPROTECT", "MS_PKGPROTECT", "NM_METHOD_NAMING_CONVENTION", "","SF_SWITCH_NO_DEFAULT","UPM_UNCALLED_PRIVATE_METHOD","URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"},
 justification = "JavaCC generated code" )
-@javax.annotation.Generated("Generated by JavaCC")
+// @javax.annotation.Generated("Generated by JavaCC") // This only works through Java 10; fails in Java 11
```

Comment:
I think these comments are for JaCoCo profiling. How do they fail in Java 11?

@pabender, do you have notes on using these lines?

Reply:
```
    [javac] /Users/jake/Documents/Trains/JMRI/projects/JMRI/java/src/jmri/jmris/srcp/parser/package-info.java:22: error: cannot find symbol
    [javac] @javax.annotation.Generated("Generated by JavaCC")
    [javac]                  ^
    [javac]   symbol:   class Generated
    [javac]   location: package javax.annotation
```

Reply:
In Java 11, that class seems to have moved:  
https://docs.oracle.com/en/java/javase/11/docs/api/java.compiler/javax/annotation/processing/Generated.html

Reply:
> I think these comments are for JaCoCo profiling. How do they fail in Java 11?
> 
> @pabender, do you have notes on using these lines?

Those were just recently added.  The idea is to mark generated code so it isn’t used for test coverage measurements.

Reply:
Does it have to be a `javax` annotation?  We could create our own, at least temporarily.